index.js

```diff
@@ -49,6 +49,13 @@ function getRandomValues (array) {
     throw new QuotaExceededError('Can only request a maximum of 65536 bytes')
   }
 
+  // Expo SDK 48+
+  if (global.expo && global.expo.modules && global.expo.modules.ExpoCrypto && global.expo.modules.ExpoCrypto.getRandomValues) {
+    // ExpoCrypto.getRandomValues doesn't return the array
```

Comment:
I think that was because we were limited in being able to return typedarrays from Expo modules back then. I'll check if we can't readd it so it matches the web spec